src/lmdb.mli

```diff
@@ -25,6 +25,12 @@ Db.put db "Bactrian camel" "Elegant and beautiful animal with two humps."
 *)
 
 
+(** {2 Capabilities} *)
+
+type 'a cap
+val ro : [ `Read ] cap
+val rw : [ `Write ] cap
```

Comment:
That's a `write-only` permission. You current `get` function will not accept it.